x-pack/plugins/apm/public/components/app/correlations/latency_correlations.tsx

```diff
@@ -5,339 +5,424 @@
+              push(history, {
+                query: {
+                  kuery: `not ${term.fieldName}:"${encodeURIComponent(
+                    term.fieldValue
+                  )}"`,
+                },
+              });
+              trackApmEvent({ metric: 'correlations_term_exclude_filter' });
             },
           },
-        });
-      }
-    },
-    [
-      environment,
-      kuery,
-      serviceName,
-      start,
-      end,
-      transactionName,
-      transactionType,
-      durationPercentile,
-      fieldNamesCommaSeparated,
-      hasFieldNames,
-      maxLatency,
-      distributionInterval,
-    ]
+        ],
+        name: i18n.translate(
+          'xpack.apm.correlations.latencyCorrelations.correlationsTable.actionsLabel',
+          { defaultMessage: 'Filter' }
+        ),
+      },
+    ],
+    [history, trackApmEvent]
   );
 
-  const trackApmEvent = useUiTracker({ app: 'apm' });
-  trackApmEvent({ metric: 'view_latency_correlations' });
+  const histogramTerms: MlCorrelationsTerms[] = useMemo(() => {
+    return histograms.map((d) => {
+      return {
+        fieldName: d.field,
+        fieldValue: d.value,
+        ksTest: d.ksTest,
+        correlation: d.correlation,
+        duplicatedFields: d.duplicatedFields,
+      };
+    });
+  }, [histograms]);
 
   return (
     <>
-      <EuiFlexGroup direction="column">
-        <EuiFlexItem>
-          <EuiText size="s" color="subdued">
-            <p>
-              {i18n.translate('xpack.apm.correlations.latency.description', {
-                defaultMessage:
-                  'What is slowing down my service? Correlations will help discover a slower performance in a particular cohort of your data. Either by host, version, or other custom fields.',
-              })}
-            </p>
-          </EuiText>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFlexGroup direction="row">
-            <EuiFlexItem>
-              <EuiTitle size="xxs">
-                <h4>
-                  {i18n.translate(
-                    'xpack.apm.correlations.latency.chart.title',
-                    { defaultMessage: 'Latency distribution' }
-                  )}
-                </h4>
-              </EuiTitle>
-              <LatencyDistributionChart
-                overallData={overallData}
-                correlationsData={
-                  hasFieldNames && correlationsData
-                    ? correlationsData?.significantTerms
-                    : undefined
+      <EuiFlexGroup data-test-subj="apmCorrelationsTabContent">
+        <EuiFlexItem style={{ flexDirection: 'row', alignItems: 'center' }}>
+          <EuiTitle size="xs">
+            <h5 data-test-subj="apmCorrelationsLatencyCorrelationsChartTitle">
+              {i18n.translate(
+                'xpack.apm.correlations.latencyCorrelations.panelTitle',
+                {
+                  defaultMessage:
+                    'Latency distribution for {name} (Log-Log Plot)',
```

Comment:
Do we want to include `Log-Log Plot` in the label? Makes it more obvious to the user I guess? If left in, should go to sentence case I think `Log-log plot` (or actually even `(log-log plot)`

Reply:
Fixed in 03e450e to just say `Latency distribution`. (One of the items in #108332 that needed a bit more clarification before I could add the change)